main.tf

```diff
@@ -1,18 +1,17 @@
 resource "aws_ram_resource_share" "subnet_sharing" {
-  count                     = length(var.private_subnet_arns) > 0 || length(var.public_subnet_arns) > 0 ? 1 : 0
```

Comment:
Are you sure this resource is _always_ needed/required? Yeah I guess it is, as otherwise why would you use this module...

Reply:
Actually that answer that you just gave is exactly why we ditched the count... it was giving us trouble and we realised that's basically all this module does, so don't use it.